DevelopGitHub/GitHubExtension/GitHubExtension.Activity.External.Tests/TestsForControllers/ActivityControllerGetActivityTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using System.Web.Http;
+using System.Web.Http.Results;
+using FluentAssertions;
+using GitHubExtension.Activity.External.Tests.Extensions;
+using GitHubExtension.Activity.External.WebAPI;
+using GitHubExtension.Activity.External.WebAPI.Controllers;
+using GitHubExtension.Activity.External.WebAPI.Models;
+using GitHubExtension.Activity.External.WebAPI.Models.EventPayloads;
+using GitHubExtension.Activity.External.WebAPI.Queries;
+using GitHubExtension.Infrastructure.Constants;
+using NSubstitute;
+using Xunit;
+
+namespace GitHubExtension.Activity.External.Tests.TestsForControllers
+{
+    public class ActivityControllerGetActivityTests
```

Comment:
As I remember we already agreed class name for our tests. It should be controller name + Tests. Can you explain what means GetActivity postfix?